plugins/MauticSocialBundle/Command/MonitorTwitterBaseCommand.php

```diff
@@ -164,48 +173,37 @@ protected function processMonitor($monitor)
         }
     }
 
-    /*
+    /**
      * Search for tweets by creating your own search criteria.
      *
-     * @return a makeRequest results array object.
+     * @return array The results of makeRequest
      */
     abstract protected function getTweets($monitor);
 
-    /*
-     * Get monitor record entity
+    /**
+     * Get monitor record entity.
      *
-     * @return \MauticPlugin\MauticSocialBundle\Entity\Monitoring $entity
+     * @param int $mid
+     *
+     * @return \MauticPlugin\MauticSocialBundle\Entity\Monitoring
      */
     protected function getMonitor($mid)
     {
-        // get the entity record
-        $entity = $this->monitoringModel->getEntity($mid);
-
-        return $entity;
+        return $this->monitoringModel->getEntity($mid);
     }
 
-    /*
-     * Processes a list of tweets and creates / updates leads in Mautic
-     *
+    /**
+     * Processes a list of tweets and creates / updates leads in Mautic.
      */
     protected function createLeadsFromStatuses($statusList, $monitor)
     {
         /** @var \Mautic\LeadBundle\Model\LeadModel $leadModel */
-        $leadModel = $this->getContainer()
-            ->get('mautic.lead.model.lead');
+        $leadModel = $this->getContainer()->get('mautic.lead.model.lead');
```

Comment:
1) Why to use container and not DI?
2) Command should be very simple and only call a service which will do the job...